### PR TITLE
[#25] Update cabal and stack.yaml files

### PR DESCRIPTION
**Pull request by rashadg1030**
In order to completely resolve issue #25, I also need to update the `travis.yml` file but I'm not sure how to go about this. The `three-layer` example repository seems to use `circleci`, so I can't really use it as a reference. Should we use circle-ci instead? Any suggestions?

### CODE REVIEW

**Comment by rashadg1030** [comment]
I fixed a typo in the commit message.

File: issue-wanted.cabal [hunk 1]
```diff
@@ -13,14 +13,17 @@ copyright:           2018 Kowainik
...
-                       IW
-                       IW.Search
+source-repository head
+  type:                git
+  location:            https://github.com/kowainik/issue-wanted.git
+
+common common-options
+  build-depends:       base-noprelude ^>= 4.12.0.0
+                     , relude ^>= 0.5.0
+                     , github >= 0.21
```
Comment (vrom911) [inline issue-wanted.cabal]: At the moment not all stanzas using `github` dependency, so it's better to add only dependencies that are used everywhere.
And also, the upper boundary is not specified. It is much better to have all the boundaries specified for the libraries.

By the way, `github-0.21` is very broken, so I would suggest using an older version. We are using `github-0.20` in another project.

File: stack.yaml [hunk 2]
```diff
@@ -1,5 +1,6 @@
-resolver: lts-12.9
+resolver: lts-13.20
```
Comment (vrom911) [inline stack.yaml]: `lts-13.21` is already out! :slightly_smiling_face:

File: issue-wanted.cabal [hunk 3]
```diff
@@ -1,9 +1,9 @@
-cabal-version:       2.2
+cabal-version:       2.4
 name:                issue-wanted
 version:             0.0.0
-description:         Web application to help beginners to start contributing into Haskell projects
+description:         Please see the README on Github at <https://github.com/kowainik/issue-wanted#readme>
```
Comment (vrom911) [inline issue-wanted.cabal]: No need to change the description on the link to README. If you feel that it's not very informative we could change it to something more useful, what do you think?

File: issue-wanted.cabal (hunk 1 above)
Comment (rashadg1030) [reply issue-wanted.cabal]: Okay got it. I will remove `github` from the common stanza. Should the boundary be like 
`github >= 0.20 && < 0.21` or just `github == 0.20`?

File: issue-wanted.cabal (hunk 3 above)
Comment (rashadg1030) [reply issue-wanted.cabal]: Got it. I think the old description is fine.

File: stack.yaml (hunk 2 above)
Comment (rashadg1030) [reply stack.yaml]: Nice! I’ll change it

File: issue-wanted.cabal (hunk 1 above)
Comment (rashadg1030) [reply issue-wanted.cabal]: I used github == 0.20

Comment (vrom911) [reply issue-wanted.cabal]: Good! Until the new non-broken version is out we can use this one. We will keep an eye on this 🙂

**Pull request merged**